apps/user-office-frontend/src/components/proposal/TableActionsDropdownMenu.tsx

```diff
@@ -0,0 +1,78 @@
+import { Menu, MenuItem } from '@mui/material';
+import React, { useEffect } from 'react';
+
+type TableActionsDropdownMenuProps = {
+  event: null | HTMLElement;
+  handleClose: (option: string) => void;
+  options: string[];
+};
+
+const TableActionsDropdownMenu: React.FC<TableActionsDropdownMenuProps> = ({
```

Comment:
React.FC is no more recommended by the community due to some of its downsides. Here is the relevant doc https://github.com/facebook/create-react-app/pull/8177. 

We also have a PR, that removes all the instances of React.FC and it will be merged soon. https://github.com/UserOfficeProject/user-office-core/pull/268


Suggested change:


`const TableActionsDropdownMenu = ({event, handleClose, options}: TableActionsDropdownMenuProps) => .............`